--- itex2MML_ruby.c.orig	2016-08-14 00:12:04.000000000 -0500
+++ itex2MML_ruby.c	2016-08-14 00:12:30.000000000 -0500
@@ -1737,7 +1737,7 @@
 SWIG_Ruby_SetModule(swig_module_info *pointer)
 {
   /* register a new class */
-  VALUE cl = rb_define_class("swig_runtime_data", rb_cObject);
+  VALUE cl = rb_define_class("Swig_runtime_data", rb_cObject);
   /* create and store the structure pointer to a global variable */
   swig_runtime_data_type_pointer = Data_Wrap_Struct(cl, 0, 0, pointer);
   rb_define_readonly_variable("$swig_runtime_data_type_pointer" SWIG_RUNTIME_VERSION SWIG_TYPE_TABLE_NAME, &swig_runtime_data_type_pointer);
